be/src/storage/rowset/struct_column_iterator.cpp

```diff
@@ -153,8 +186,8 @@ Status StructColumnIterator::seek_to_first() {
     if (_null_iter != nullptr) {
         RETURN_IF_ERROR(_null_iter->seek_to_first());
     }
-    for (auto& iter : _field_iters) {
-        RETURN_IF_ERROR(iter->seek_to_first());
+    for (int i = 0; i < _field_iters.size(); ++i) {
```

Comment:
why change this?

Reply:
sorry, it's a useless modification, I will rollback it